src/types/feeBreakdown.ts

```diff
@@ -2,13 +2,22 @@ export interface FeeBreakdown {
   amount: {
     calculated: number;
     payable: number;
-    reduction: number;
     vat: number;
-  };
+  } & ReductionOrExemption;
```

Comment:
This FeeBreakdown type is really helpful, but I do find a bit confusing that all properties appear required - whereas my instinct is that everything besides `payable` should have a `?` ?

Reply:
The reason for this is that I'm explicitly defaulting values to 0 when parsing the passport. 

It felt more explicit to me to have properties such as `vat: 0` as opposed to `vat: undefined`. The intention would be to say "we've checked this and it's zero" vs "this isn't included".

Functionally, as 0 is falsy we can still write `if (!feeBreakdown.vat)` etc.

Reply:
Thanks for clarification here! 

Let's keep thinking on the language for this - for example my instinct is that "we've checked this and _it doesn't apply_ (is undefined)" is desirable (and how we'd think about this for schema validation), not "we've checked this and it's zero" (how you think about for fee breakdown math) - it's tricky !!